python/pyspark/sql/pandas/group_ops.py

```diff
@@ -454,7 +582,7 @@ def applyInPandas(
         the grouping key(s) will be passed as the first argument and the data will be passed as the
         second and third arguments.  The grouping key(s) will be passed as a tuple of numpy data
         types, e.g., `numpy.int32` and `numpy.float64`. The data will still be passed in as two
-        `pandas.DataFrame` containing all columns from the original Spark DataFrames.
+        `pandas.DataFrame`\\s containing all columns from the original Spark DataFrames.
```

Comment:
nit: We don't need to escape the letter "s" in docstrings

Reply:
Removed this unrelated change.